README.md

```diff
@@ -58,3 +58,22 @@ command:
 ```
 firefox com.ibm.streamsx.metrics/doc/spldoc/html/index.html &
 ```
+
+### Building Project
+The MetricsSink operator requires you to build jars and include them in your project. You will also need to have an ElasticSearch server running.
```

Comment:
Can you provide some links that help to setup the ElasticSearch server? And what is with the InfluxDB?